packages/perseus/src/perseus-types.ts

```diff
@@ -690,6 +690,15 @@ export type LockedLineType = {
     showPoint2: boolean;
 };
 
+export type LockedCircleType = {
+    type: "circle";
+    center: Coord;
+    radius: number;
+    color: LockedFigureColor;
+    fillOpacity: number;
```

Comment:
🤔  I don't know if there's a requirement to give the authors the ability to control _how_ translucent the fill is.  What do you think about just having `boolean` for this?  Like `opaqueFill: boolean` or `translucent: boolean`?

Reply:
🤔  Thinking a bit further.  Maybe it's two flags:

```js
  filled: boolean,
  translucentFill: boolean,
```

What do you think?

Reply:
Yep!  That's great!!